### PR TITLE
Add cli args

### PR DESCRIPTION
**Pull request by jakoivis**
Added couple of cli args that we would need. I saw you had plans for these and tags was also mentioned on one ticket. 
- tags: to run only some tagged tests. I don't know about cucumber enough to say whether this is a good solution or not, but seems to work pretty well.
- concurrency, app and appInitDelay

### CODE REVIEW

File: src/args.js [hunk 1]
```diff
@@ -54,4 +54,26 @@ module.exports = require('yargs')
...
+    describe: 'Specifies that tests should run concurrently',
+    type: 'number'
+  })
+  .option('app', {
+    alias: 'a',
+    default: null,
+    describe: 'Executes the specified shell command before running tests. Use it to launch or deploy the application you are going to test.',
+  })
+  .option('appInitDelay', {
+    default: 1000,
```
Comment (Lukas-Kullmann) [inline src/args.js]: I think the default should be `0` or `undefined` since it is optional in the original API.

File: src/runner.js [hunk 2]
```diff
@@ -11,16 +11,25 @@ module.exports = async ({
...
+  speed,
+  concurrency,
+  app,
+  appInitDelay,
+  tags
 }) => {
   const testcafe = await createTestCafe(hostname, ...ports.slice(0, 2));
   const runner = testcafe.createRunner();
 
+  console.log("Running with browsers", browsers, "tags", tags, "concurrency", concurrency);
```
Comment (Lukas-Kullmann) [inline src/runner.js]: I don't like this. The information seems to be redundant to me.

File: src/TestcafeGherkinBootstrapper.js [hunk 3]
```diff
@@ -132,4 +138,22 @@ module.exports = class TestcafeGherkinBootstrapper extends TestcafeBootstrapper
       await this._runStep(hook.code, testController, []);
     }
   }
+
+  _shouldRunScenario(scenario) {
+    if (!this.tags || !this.tags.length) {
```
Comment (Lukas-Kullmann) [inline src/TestcafeGherkinBootstrapper.js]: How can `'this.tags` be falsy?

File: src/args.js (hunk 1 above)
Comment (jakoivis) [reply src/args.js]: The value 1000 was used here https://github.com/DevExpress/testcafe/blob/master/src/runner/bootstrapper.js but I think 0 is fine as well.

File: src/runner.js (hunk 2 above)
Comment (jakoivis) [reply src/runner.js]: Yep, I forgot to remove that

File: src/TestcafeGherkinBootstrapper.js (hunk 3 above)
Comment (jakoivis) [reply src/TestcafeGherkinBootstrapper.js]: I guess it cannot be. I'll verify and remove it.

File: README.md [hunk 4]
```diff
@@ -48,6 +48,10 @@ The following options are supported:
 | debugMode | N.A. | Optional | Execute test steps one by one pausing the test after each step | false |
 | debugOnFail | N.A. | Optional | Enter debug mode when the test fails | false |
 | speed | N.A. | Optional | Set the speed of test execution (0.01 ... 1) | 1 |
+| concurrency | c | Optional | Specifies that tests should run concurrently | 1 |
+| app | a | Optional | Executes the specified shell command before running tests. Use it to launch or deploy the application you are going to test. | null |
+| appInitDelay | N.A. | Optional | Specifies the time (in milliseconds) allowed for an application launched using the --app option to initialize. | 0 |
+| tags | t | Optional | Run only tests having the specified tags | [] |
```
[the remark this answers is not in the document]
Comment (Lukas-Kullmann) [reply README.md]: I think the default values that you provided are fine. :)

**Comment by Lukas-Kullmann** [comment]
Thanks for contributing, @jakoivis!
I just published v1.3 to npm. :)

**Pull request merged**